packages/terra-list/src/ListSectionHeader.module.scss

```diff
@@ -86,4 +86,17 @@
       transform: rotate(90deg);
     }
   }
+
+  // List Item Component
+  // ========================================================
+  .list-item {
+    align-items: center;
+    background-color: var(--terra-list-item-background-color);
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+    min-height: var(--terra-list-item-min-height);
+    padding: 0;
+    width: 100%;
+  }
```

Comment:
this class is already defined on list.module.scss !!

Reply:
ListSectionHeader only imports ListSectionHeader.module.scss https://github.com/cerner/terra-core/blob/f19aef99b61648edb2b5bfa94028db8954ec9c48/packages/terra-list/src/ListSectionHeader.jsx#L7